index.bs

```diff
@@ -10018,35 +10140,60 @@ interface AudioWorkletProcessor {
 
 <h5 id="AudioWorketProcessor-constructors">
 Constructors</h5>
+
 <dl dfn-type="constructor" dfn-for="AudioWorkletProcessor">
 	: <dfn>AudioWorkletProcessor(options)</dfn>
 	::
-		When the constructor for {{AudioWorkletProcessor}} is invoked, the following steps are performed:
+		When the constructor for {{AudioWorkletProcessor}} is invoked,
+		the following steps are performed on the <a>rendering thread</a>.
+
```

Comment:
This is basically a move of the existing algorithm from another section to this, right? No other changes?

Reply:
It depends on what your "this" means here. This PR has few more changes integrated. I summarized them in the description.

But if you're asking about this L.10147 specifically - Yes. It's a simple move.